sqlparser/lib/src/analysis/schema/view.dart

```diff
@@ -0,0 +1,104 @@
+part of '../analysis.dart';
+
+/// A database view. The information stored here will be used to resolve
+/// references and for type inference.
+class View with HasMetaMixin implements HumanReadable, ResolvesToResultSet {
+  /// The name of this table, as it appears in sql statements. This should be
+  /// the raw name, not an escaped version.
+  ///
+  /// To obtain an escaped name, use [escapedName].
+  final String name;
+
+  /// If [name] is a reserved sql keyword, wraps it in double ticks. Otherwise
+  /// just returns the [name] directly.
+  String get escapedName {
+    return isKeywordLexeme(name) ? '"$name"' : name;
+  }
+
+  final List<ViewColumn> resolvedColumns;
+
+  /// Filter the [resolvedColumns] for those that are
+  /// [Column.includedInResults].
+  List<ViewColumn> get resultColumns =>
+      resolvedColumns.where((c) => c.includedInResults).toList();
+
+  /// Whether this table was created with an "WITHOUT ROWID" modifier
+  final bool withoutRowId;
+
+  /// Additional constraints set on this table.
+  final List<TableConstraint> tableConstraints;
+
+  /// The ast node that created this table
+  final TableInducingStatement definition;
```

Comment:
This should be a `CreateViewStatement`, accordingly.

Reply:
whoops, you're right, fixed :laughing: